forc/src/cli/commands/build.rs

```diff
@@ -14,10 +14,10 @@ pub struct Command {
     #[clap(short, long)]
     pub path: Option<String>,
     /// Whether to compile using the original (pre- IR) pipeline.
-    #[clap(long)]
+    #[clap(long, hide = true)]
     pub use_orig_asm: bool,
     /// Whether to compile using the original (pest based) parser.
-    #[clap(long)]
+    #[clap(long, hide = true)]
```

Comment:
I think you need to do exactly the same thing in `run.rs` and `deploy.rs` too.  The issue only mentioned `build` but it makes sense to hide them everywhere.

Reply:
makes sense, thanks for the catch!

Reply:
Wait, does `forc test` not have options to turn off these flags?? 😂

Reply:
`forc test` only builds the Rust test harness and not the Sway code, right? It can't tell the compiler what to do.

Reply:
https://github.com/FuelLabs/sway/blob/07d36690e6478894fc271b8a49cb9a771ddb552a/forc/src/cli/commands/test.rs#L44-L45

Reply:
Maybe leave for a subsequent PR @binggh

Reply:
TIL. I always thought `forc test` was just an alias for `cargo test` (and I've always used `cargo test`) but I guess it's doing more things now.

Reply:
I believe the default values used in Default::default() are `false`, so the flags won't be set by default i think: https://doc.rust-lang.org/src/core/default.rs.html#187